api/src/controllers/sync.ts

```diff
@@ -80,7 +80,9 @@ export const getProvisionedProfileBotJson = async (
       throw new Error(errmsg);
     }
 
-    const context = await contextForProvisioning(profileId, true);
+    const clusters = await NamespaceModel.findClustersForProfile(profile.id);
+
+    const context = await contextForProvisioning(profileId, clusters[0], true);
```

Comment:
Is zero a magic number? What's so special about it? If there is only one do a pop(). I think the mystery is why not use 1 or what happens if I do.